product/tasks.md

```diff
@@ -71,13 +71,13 @@ The `x:size/<value>` labels describe the expected amount of work for a contribut
 
 The `x:type/<value>` labels describe the type of work the contributor will be engaged in.
 
-| Tag              | Description                                                       |
-| ---------------- | ----------------------------------------------------------------- |
-| `x:type/ci`      | Work on Continuous Integration (e.g. GitHub Actions workflows)    |
-| `x:type/coding`  | Work on code that is not part of the content (e.g. track tooling) |
-| `x:type/content` | Work on content (e.g. exercises, concepts)                        |
-| `x:type/docker`  | Work on Dockerfiles                                               |
-| `x:type/docs`    | Work on Documentation                                             |
+| Tag              | Description                                                                                                                                                                                                                     |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `x:type/ci`      | Work on Continuous Integration (e.g. GitHub Actions workflows)                                                                                                                                                                  |
+| `x:type/coding`  | Work involving writing code that is not part of the student-facing content. This is normally tasks associated with production-tooling (e.g. test runners), internal-tooling (generators), or libraries that Exercism maintains. |
```

Comment:
```suggestion
| `x:type/coding`  | Write code that is not part of the student-facing content. This is normally tasks associated with production-tooling (e.g. test runners), internal-tooling (generators), or libraries that Exercism maintains. |
```

Reply:
(I realised this fits better with how the others are worded)

Reply:
(I can't apply suggestion directly via GH because formatting) [`e663676` (#152)](https://github.com/exercism/docs/pull/152/commits/e6636768cc5eeb2ed7d7b1ed2ede2a2b6c87075c)